dist/jquery-dateFormat.js

```diff
@@ -463,6 +472,12 @@ var DateFormat = {};
       },
       toBrowserTimeZone : function(value, format) {
         return this.date(new Date(value), format || 'MM/dd/yyyy HH:mm:ss');
+      },
+      toTimeZone : function (value, timeZone, format){
+        // timeZone : Asia/Tokyo etc..
+        // clear LRM in IE
+        var date = new Date(value).toLocaleString('en-US', {timeZone: timeZone, hour12: false}).replace(/&lrm;|\u200E/gi, '');
+        return this.date(new Date(date), format || 'MM/dd/yyyy HH:mm:ss');
```

Comment:
@nev-tuannm4 could you move the change to https://github.com/phstc/jquery-dateFormat/blob/master/src/dateFormat.js, this `dist/jquery-dateFormat.js` is the compiled file.